File: Pohon Natal/Java/pohon-natal-i.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args) {
	    
	    Scanner userInput = new Scanner(System.in);
    	
        System.out.print("Masukan Tinggi Pohon : ");
        int x = userInput.nextInt();
        for(int i = 1; i <= x; i++){
            for(int j = x; j >= i; j--){
                System.out.print(" ");
            }
            
            for(int l = 2; l <= i; l++){
               System.out.print("*");
            }
            
            for(int k = 1; k <= i; k++){
                    System.out.print("*");
                }
                
            System.out.println("");
                
            if(i==x){
                for(int m = 0; m < 3; m++){
                    for(int o = 1; o < i; o++){
                    System.out.print(" ");
                    }
                    
                    for(int n = 0; n <= 0; n++){
                    System.out.print("===");
                    }
                    
                System.out.println("");
                }
            }
        }
	}
}

File: Pohon Natal/GO/pohon-natal-i.go
package main
import "fmt"

func main(){
    var x int
    fmt.Print("Masukan Tinggi Pohon: ")
    fmt.Scan(&x)
    
    for i := 1; i <= x; i++ {
        for j := x; j >= i; j-- {
            fmt.Print(" ")
        }
        
        for l := 2; l <= i; l++ {
            fmt.Print("*")
        }
        
        for k := 1; k <= i; k++ {
            fmt.Print("*")
        }
            
        fmt.Print("\n")
            
        if i==x {
            for m := 0; m < 3; m++ {
                for o := 1; o < i; o++ {
                    fmt.Print(" ")
                }
                
                for n := 0; n <= 0; n++ {
                    fmt.Print("===")
                }
                fmt.Print("\n")
            }
        }
    }
}
